--- libvideogfx-1.0.9/libvideogfx/graphics/fileio/ffmpeg.cc.orig	2015-10-15 21:48:26.217193172 +0200
+++ libvideogfx-1.0.9/libvideogfx/graphics/fileio/ffmpeg.cc	2015-10-15 21:50:12.237190962 +0200
@@ -166,7 +166,7 @@
     if (frameRGB)  { av_free(frameRGB); frameRGB=NULL; }
     if (frame)     { av_free(frame); frame=NULL; }
     if (codecCtx)  { avcodec_close(codecCtx); codecCtx=NULL; }
-    if (formatCtx) { av_close_input_file(formatCtx); formatCtx=NULL; }
+    if (formatCtx) { avformat_close_input(&formatCtx); formatCtx=NULL; }
   }
 
 
